ui/search/nuxeo-results-view.js

```diff
@@ -0,0 +1,453 @@
+            </iron-collapse>
+            <div class="buttons" hidden$="[[!opened]]">
+              <paper-button noink class="clear secondary" on-tap="_clear" disabled$="[[loading]]">
+                [[i18n('command.clear')]]
+              </paper-button>
+              <div class="horizontal layout flex end-justified">
+                <paper-button noink class="primary search" on-tap="_search" hidden$="[[auto]]" disabled$="[[loading]]">
+                  [[i18n('command.search')]]
+                  <template is="dom-if" if="[[loading]]">
+                    <paper-spinner-lite active></paper-spinner-lite>
+                  </template>
+                </paper-button>
+              </div>
+            </div>
+          </div>
+        </template>
+
+        <template is="dom-if" if="[[visible]]">
+          <nuxeo-search-results-layout
+            id="results"
+            href-base="[[hrefBase]]"
+            search-name="[[searchName]]"
+            nx-provider="[[_nxProvider]]"
+            on-navigate="_navigateFromSearch"
+            on-results-changed="_resultsChanged"
+          ></nuxeo-search-results-layout>
+        </template>
+      `;
+    }
+
+    static get is() {
+      return 'nuxeo-results-view';
+    }
+
+    static get properties() {
+      return {
+        /**
+         * The id of the `nuxeo-page-provider` instance used to perform the search.
+         */
+        provider: {
+          type: String,
+        },
+        /**
+         * The number of results per page.
+         */
+        pageSize: {
+          type: Number,
+          value: 40,
+        },
+        /**
+         * The query parameters passed on to `provider`.
+         */
+        params: {
+          type: Object,
+          observer: '_paramsChanged',
+          value: {},
+        },
+        /**
+         * List of content enrichers passed on to `provider`.
+         * Already set by default are thumbnail, permissions and highlight.
+         */
+        enrichers: {
+          type: String,
+          value: 'thumbnail, permissions, highlight',
+        },
+        /**
+         * The headers passed on to `provider`.
+         * Already set by default are 'fetch-document': 'properties' and 'translate-directoryEntry': 'label'.
+         */
+        headers: {
+          type: String,
+          value: { 'fetch-document': 'properties', 'translate-directoryEntry': 'label' },
+        },
+        /**
+         * List of comma separated values of the document schemas to be returned.
+         * All document schemas are returned by default.
+         */
+        schemas: {
+          type: String,
+        },
+        /**
+         * If `true`, requests from `provider` are in flight.
+         */
+        loading: {
+          type: Boolean,
+          reflectToAttribute: true,
+          value: false,
+        },
+        /**
+         * The name of the search layout.
+         */
+        searchName: String,
```

Comment:
same remark, can this have a default based on the provider ?

Reply:
For `<nuxeo-results-view>` maybe we should keep its API as is, as it might be used already in some custom configuration (specially since it is used by Designer to scaffold tab listings, for example).